root/static/styles/layout.less

```diff
@@ -1018,6 +1018,10 @@ a.tagger-icon {
     width: 1px;
 }
 
+.number-column {
+    width: 1em;
```

Comment:
Seems a bit cramped since even something like "TH 5" wraps.

Reply:
True, but it's what it already had. Wonder what would make more sense - do like with .actions and have it auto-wrap to as small as needed?

Reply:
Yeah I realized after that you were just moving it here. Not sure honestly, CSS is hard. It's probably worth figuring out as a separate ticket.